rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2245,6 +2245,38 @@ namespace rsx
 		}
 	}
 
+	void thread::delay_fifo(u64 div)
```

Comment:
For large values, something better needs to be done; for example linux has timerfds with fairly decent accuracy in the > 15us range. Some refactoring should be considered for this.
Also maybe name the function fifo_wake_delay() to convey purpose. Other than that nit, the rest looks fine, default settings seem to be a perf match for master which has a 1-2us wake delay.